pkg/pillar/zedcloud/send.go

```diff
@@ -133,11 +134,19 @@ func SendOnAllIntf(ctx *ZedCloudContext, url string, reqlen int64, b *bytes.Buff
 // Otherwise we test non-free interfaces also.
 // We return a bool remoteTemporaryFailure for the cases when we reached
 // the controller but it is overloaded, or has certificate issues.
+// Return Values:
+//    success/Failure, remoteTemporaryFailure, error
+//      If Failure,
+//          remoteTemporaryFailure indicates if it is a temp. one
```

Comment:
"remote" is more important than "temporary", and you need to fix the code based on the old comment below.

Reply:
I did not change this part of the code.. Or was this meant for the older version of the code?

Reply:
Per the diff you clearly added this comment, and the comment is misleading.
A better comment would be
 remoteTemporaryFailure indicates the failure is with the remote controller (such as an expired certificate etc)

Reply:
"you need to fix the code based on the old comment below." --> THis threw me off - thought u were referring to the code..

Ah.. I see what u mean.. 

"remoteTemporaryFailure indicates if it is a temp." --> This seems to give an indication it is a temporary failure as opposed to "Remote Failure" aka.. something like - the controller is being upgraded etc.. 

As an aside - Why do we have "Temporary" in there? There is no "RemotePermanentFailure" - right? So it just means "remoteFailure" --> Which means - retry for Eve - Retry in hope of remote end point getting fixed soon - right?

Anyways.. changed the comment as:

If Failure,
    remoteTemporaryFailure indicates if it is a remote failure.